tests/cypress/integration/skeleton/skeletons_import_export.js

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) 2022 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Importing and exporting skeletons', () => {
+    const fileToUpload = 'mounted_file_share/svg/upload.svg';
+    const { downloadsFolder } = require('../../../cypress.json');
+
+    before(() => {
+        cy.task('createFolder', downloadsFolder);
+    });
+
+    beforeEach(() => {
+        cy.loginHeadless();
+    });
+
+    after(() => {
+        cy.task('cleanFolder', downloadsFolder);
+    });
+
+    describe('Import and export svg skeleton file', () => {
+        // TODO Move to common utils
+        const getSvgElementCount = (filePath) => cy.readFile(filePath)
+            .then((content) => {
+                const contentWithRoot = `<root>${content}</root>`;
+                const parser = new DOMParser();
+                const xml = parser.parseFromString(contentWithRoot, 'image/svg+xml');
+                /* eslint-disable-next-line */
+                expect(xml.querySelector('parsererror'), 'SVG parsing error').to.be.null;
+                const circles = xml.getElementsByTagName('circle');
+                const lines = xml.getElementsByTagName('line');
+                console.log({
+                    circle: circles.length,
+                    line: lines.length,
+                });
+                return {
+                    circle: circles.length,
+                    line: lines.length,
+                };
+            });
+
+        // TODO Move to common utils
+        const compareCanvasWithSvg = (svgFilePath) => {
+            getSvgElementCount(svgFilePath).then((svgElements) => {
+                cy.get('.cvat-skeleton-configurator-svg').within(() => {
+                    cy.get('circle').should('have.length', svgElements.circle);
+                    if (svgElements.line) {
+                        cy.get('line').should('have.length', svgElements.line);
+                    } else {
+                        cy.get('line').should('not.exist');
+                    }
+                });
+            });
+        };
```

Comment:
In general would be better to recursively compare documents structure, I believe. Not just number of elements.